File: app/src/main/java/eu/pkgsoftware/babybuddywidgets/Tools.java
package eu.pkgsoftware.babybuddywidgets;

import android.content.Context;

public class Tools {
    public static int dpToPx(Context context, float dp) {
        return (int) (context.getResources().getDisplayMetrics().density * dp + 0.5f);
    }
}

File: app/src/main/java/eu/pkgsoftware/babybuddywidgets/Tools.kt
package eu.pkgsoftware.babybuddywidgets

import android.content.Context

object Tools {
    @JvmStatic
    fun dpToPx(context: Context, dp: Float): Int {
        return (context.resources.displayMetrics.density * dp + 0.5f).toInt()
    }
}